### ISSUE TITLE
Which files need to be hosted?

### ISSUE THREAD

**Issue by pvnr0082t** [liferay/alloy-editor#598]
Hi @ipeychev ,
We are [cdnjs](https://github.com/cdnjs/cdnjs) team, we want to host this project in cdnjs.
But I'm not sure which files in this repo need to be added, could you please help up find out the main files?
Thank you.

https://github.com/cdnjs/cdnjs/issues/8199

**Comment by ipeychev** [comment]
Hello,

Thanks for the hosting offer. 

TL;DR Is it possible to host the project structure as it is [here](https://github.com/liferay/alloy-editor/tree/master/dist/alloy-editor)? The .md files are not needed, of course.

Explanations:
The project's dist files are [here](https://github.com/liferay/alloy-editor/tree/master/dist/alloy-editor). It doesn't have a single possible JS dist file, but multiple ones and the developers load on the pages what is suitable for their projects. For example, some people may have CKEditor already, so they will load the file without CKEditor, other may have React, so they need another one and so on. Minimized files or not is another option.
The same is valid for the CSS files - developers choose which one depending on the theme they want to use.
On top of that, there are files, loaded dynamically. We have ~68 languages files, but only one of them is loaded on the page and it is loaded dynamically, on the fly (we determine the most appropriate one runtime, checking browser's language).
CKEditor's engine loads one JS file dynamically too.
Do you host CKEditor on the cdnjs? If so, the hosting should be very similar to theirs.

Thanks,

Thanks

**Comment by pvnr0082t** [comment]
Yes, we host [ckeditor](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/ckeditor) on cdnjs. Thanks for your explanation.
So what you mean is all the files under `dist` folder need to be added in cdnjs except `.md` files, right?

Thanks.

**Comment by ipeychev** [comment]
Yes, correct.

Thanks,

**Comment by jbalsas** [comment]
Hey @pvnr0082t, do you still need something to make this happen? Also, is there something we need to do to make sure new versions are hosted as well? We just released `1.2.5`.

**Comment by pvnr0082t** [comment]
@jbalsas Thanks for reminding. I'll try my best. If there is any problem to do this, I'll ask for help. :smile:

**Comment by jbalsas** [comment]
Sure! Let us know if we can help in any way!

**Comment by wincent** [comment]
Closing due to inactivity.
